CMakeLists.txt

```diff
@@ -401,8 +401,8 @@ if (COMPILER_CLANG)
 
     # Set new experimental pass manager, it's a performance, build time and binary size win.
     # Can be removed after https://reviews.llvm.org/D66490 merged and released to at least two versions of clang.
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fexperimental-new-pass-manager")
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fexperimental-new-pass-manager")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fexperimental-new-pass-manager -mllvm -inline-threshold=10000")
```

Comment:
Is the inlining threshold related to the experimental pass manager? Guess not, but it looks so --> maybe set the threshold in a separate SET(CMAKE_CXX_FLAGS ...) statement?

Also, it would be nice to add a brief comment a la "increment inlining threshold to increase the chance of inlining".

Reply:
It is experimental setup it will not be merged.